python/src/otel/otel_sdk/otel-instrument

```diff
@@ -40,5 +43,67 @@ environ["OTEL_RESOURCE_ATTRIBUTES"] = "%s,%s" % (
     environ.get("OTEL_RESOURCE_ATTRIBUTES"),
 )
 
-# start the runtime with the extra options
+import logging
+
+from opentelemetry.environment_variables import OTEL_PYTHON_DISABLED_INSTRUMENTATIONS
+from opentelemetry.instrumentation.aws_lambda import AwsLambdaInstrumentor
+from pkg_resources import iter_entry_points
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def _load_distros():
+    for entry_point in iter_entry_points("opentelemetry_distro"):
+        try:
+            entry_point.load()().configure()  # type: ignore
+            logger.info("Distribution %s configured", entry_point.name)
+        except Exception as exc:  # pylint: disable=broad-except
+            logger.info("Distribution %s configuration failed", entry_point.name)
+
+
+def _load_instrumentors():
+    package_to_exclude = environ.get(OTEL_PYTHON_DISABLED_INSTRUMENTATIONS, [])
+    if isinstance(package_to_exclude, str):
+        package_to_exclude = package_to_exclude.split(",")
+        # to handle users entering "requests , flask" or "requests, flask" with spaces
+        package_to_exclude = [x.strip() for x in package_to_exclude]
+    for entry_point in iter_entry_points("opentelemetry_instrumentor"):
+        try:
+            if entry_point.name in package_to_exclude:
+                logger.info("Instrumentation skipped for library %s", entry_point.name)
+                continue
+            entry_point.load()().instrument()  # type: ignore
+            logger.info("Instrumented %s", entry_point.name)
+        except Exception as exc:  # pylint: disable=broad-except
+            logger.info("Instrumenting of %s failed", entry_point.name)
+
+
+def _load_configurators():
+    configured = None
+    for entry_point in iter_entry_points("opentelemetry_configurator"):
+        if configured is not None:
+            logger.warning(
+                "Configuration of %s not loaded, %s already loaded",
+                entry_point.name,
+                configured,
+            )
+            continue
+        try:
+            entry_point.load()().configure()  # type: ignore
+            configured = entry_point.name
+        except Exception as exc:  # pylint: disable=broad-except
+            logger.info("Configuration of %s failed", entry_point.name)
+
+
+_load_distros()
+_load_configurators()
+_load_instrumentors()
+
+# TODO: Move to opentelemetry-python-contrib so that adding
+# `opentelemetry-instrumentation-lambda` to the requirements.txt file will let
+# this automatically be picked up by `_load_instrumentors()` above
+AwsLambdaInstrumentor().instrument(skip_dep_check=True)
+
+# Start the runtime with the originally intended arguments
 system(" ".join(args))
```

Comment:
When asking if this is intended, got a reply from @wangzlei :

> Yes, I copied system(" ".join(args)) from Lambda python example, you can modify it.

and

> args is sys.argv[1:], should be same. Follow up from [a previous PR](https://github.com/open-telemetry/opentelemetry-lambda/pull/124#discussion_r680089874)

I think we should just do `system(" ".join(sys.argv[1:]))` but I'm concerned this will ignore edits to the `argv` commands made in the code above? But I also don't think there will be any edits... have to confirm.